wrappers/opencv/face/readme.md

```diff
@@ -0,0 +1,13 @@
+# rs-imshow Sample
+
+## Overview
+This example is a "hello-world" code snippet for Intel RealSense cameras integration with OpenCV. The sample will open an OpenCV UI window and render colorized depth stream to it. 
```

Comment:
??

Reply:
This file needs to be taken out of the changes -- the opencv/face/ example is not going in...